redi-connect-front/src/components/organisms/RediHero.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+  Container,
+  Section,
+  Hero,
+  Columns,
+  Heading,
+  Content,
+} from "react-bulma-components";
+import Button from "../atoms/Button";
+import team from "../../assets/images/hero.svg";
+import teamMobile from "../../assets/images/hero-mobile.svg";
+import "./RediHero.scss";
+
+const RediHero = () => (
+  <Hero>
+    <Container>
+      <Section>
+        <Columns>
+          <Columns.Column size={5} className="hero__column">
+            <Heading size={1} renderAs="h1" className="hero__column--heading">
+              Welcome to ReDI Connect{" "}
+            </Heading>
+            <Columns.Column className="is-hidden-tablet">
```

Comment:
This is not a big problem, but it may be a bit more secure to use the props for rendering out the classes. This way you are just hardcodeing the classes and its much easier to make a typo...

Reply:
I removed it, it's H1 by default